model/Model.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  firstname: { type: String, required: false },
+  middlename: { type: String, required: false },
+  surname: { type: String, required: false },
+  dob: { type: Date, required: false },
+  address: { type: String, required: false },
+  sex: { type: String, required: false },
+  isActive: { type: Boolean, required: false },
+  phone: { type: Number, required: false },
+  state: { type: String, required: false },
+  religion: { type: String, required: false },
+  maritalStatus: { type: String, required: false },
```

Comment:
nit: So, `maritalStatus: { type: String` and `username:{type:String` look a bit inconsistent. I think we should go with `maritalStatus: { type: String` style since it's the standardjs recommend style. And it looks cleaner. What do you think?

Reply:
I think after updating this we can merge

Reply:
Actually, I was refering to the space in between the curly braces and the colon. the one with space is prefered according to standardjs that we use.

Reply:
you mean to remove the required property?

Reply:
No, I meant this:
> Actually, I was refering to the space in between the curly braces and the colon. the one with space is prefered according to standardjs that we use.